test/generators/javascript/preset/__snapshots__/MarshallingPreset.spec.ts.snap

```diff
@@ -0,0 +1,165 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Marshalling preset should render un/marshal code 1`] = `
+"class Test {
```

Comment:
Try copy paste this output into a `.js` file and see if it works 🙂

Reply:
I have made a file named `MarshallingPreset.js` and copied everything present inside `test/generators/javascript/preset/__snapshots__/MarshallingPreset.spec.ts.snap` (removed export part) and executed `npm run test`  but I am not sure if it is the right way of doing it or have I done anything wrong here, kindly help me out.

Reply:
`npm run test` only executes tests i.e. files with the ending `x.spec.ts`, to execute the copied content you could just do `node MarshallingPreset.js`, your IDE, should also give you the problems.

Let me know if that helped 🙂